geode-apis-compatible-with-redis/src/test/java/org/apache/geode/redis/internal/data/RedisHashTest.java

```diff
@@ -258,11 +261,380 @@ public void hscanSnaphots_shouldExpireAfterExpiryPeriod() {
     });
   }
 
+  /************* Hash Size *************/
+  /******* constants *******/
+  @Test
+  public void constantBaseRedisHashOverhead_shouldEqualCalculatedOverhead() {
+    ReflectionObjectSizer reflectionObjectSizer = ReflectionObjectSizer.getInstance();
+    int baseRedisHashOverhead = reflectionObjectSizer.sizeof(new RedisHash());
+
+    assertThat(baseRedisHashOverhead).isEqualTo(BASE_REDIS_HASH_OVERHEAD);
+  }
```

Comment:
This test is the same as `should_calculateSize_equalToROSSize_ofEmptyIndividualInstance()`.  Do we need both? Also, it looks like we already have the class variable `reflectionObjectSizer` that we could use instead of getting another instance.

Reply:
they test slightly different things but since they have a lot of overlapping code, I put both assertions in this test and deleted the other one

Reply:
ah ok, I see that now. If we want to keep both that makes sense.